packages/fuels-test-helpers/Cargo.toml

```diff
@@ -14,6 +14,7 @@ fuel-crypto = "0.5"
 fuel-gql-client = { version = "0.7", default-features = false }
 fuel-types = { version = "0.5", default-features = false }
 fuels-signers = { version = "0.13.0", path = "../fuels-signers", optional = true }
+fuels-contract = { version = "0.13.0", path = "../fuels-contract" }
```

Comment:
I'm not a big fan of adding the `contracts` workspace as a dep, it seems like a lot. @digorithm  what do you think? I'm not very well-versed in alternative solutions.

Reply:
I have not found another way to import fuel-contract. I could import it into some of the dependencies I already use and then use the fuel-contract through them, but in the end it's the same as this. Any suggestions?

Reply:
I was scared of potential circular deps but it seems ok for now.